src/main/scala/uk/gov/homeoffice/drt/service/CiriumService.scala

```diff
@@ -27,7 +28,7 @@ class CiriumService[F[_] : Sync](airlineConfig: AirlineConfig, client: Client[F]
           if (r.status == Status.Ok || r.status == Status.Conflict) {
```

Comment:
Could we rename `process` to something more descriptive. I presume it's fetching missing departure times from the arrivals it's passed. If that's what it's doing then maybe call it something like `populateMissingDepartureTimes` or `fetchMissingDepartureTimes` if it's not actually updating anything (it looks like it returns the cirrium response)

Reply:
completely agree